### PR TITLE
PBI 50675 implement play/pause function with animation

### PR DESCRIPTION
**Pull request by patriciabarrosssw**

### CODE REVIEW

**Comment by wicksipedia** [comment]
Hey @patriciabarrosssw it looks like the profile box and person components are getting a lot of audio concerns now.

I'd recommend extracting the audio ui and its effects into a separate component

File: src/components/play-audio/play-audio.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                playAudio();
+              }}
+            />
+          )}
+          {isPlaying && (
+            <PauseIcon
+              aria-label="pause audio"
+              className={'cursor-pointer mr-2'}
+              onClick={() => {
+                stopAudio();
```
Comment (gabrielgeorge) [inline src/components/play-audio/play-audio.js:70]: Does this pause actually stop the audio and revert the current time back to 0?

Comment (patriciabarrosssw) [reply src/components/play-audio/play-audio.js]: You are right, I shouldn't be reverting the audio current time back to 0.... I changed the function name and it just pauses the audio.

**Comment by patriciabarrosssw** [comment]
> Hey @patriciabarrosssw it looks like the profile box and person components are getting a lot of audio concerns now.
> 
> I'd recommend extracting the audio ui and its effects into a separate component

Play/Pause Audio now is a component.... Thanks for the feedback.

File: src/components/play-audio/play-audio.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect, createRef } from 'react';
+import PlayIcon from '-!svg-react-loader!../../images/SSWPlay2x.svg';
+import PauseIcon from '-!svg-react-loader!../../images/SSWPause2x.svg';
+import lottie from 'lottie-web';
```
[the remark this answers is not in the document]
Comment (patriciabarrosssw) [reply src/components/play-audio/play-audio.js:4]: I added with yarn add lottie-web and it came from https://github.com/airbnb/lottie-web

[the remark this answers is not in the document]
Comment (wicksipedia) [reply src/components/play-audio/play-audio.js:4]: I've added it again through yarn and it updated the package.json

**Pull request merged**